old-fashioned-image-build

```diff
@@ -177,6 +181,11 @@ time for FILE in `lxc exec lp-$SERIES-${ARCH} -- find /build -mindepth 1 \
     lxc file pull lp-$SERIES-${ARCH}$FILE $OUTPUT_DIRECTORY/
 done
 
+if [ $CLEANUP = false ] ; then
+    echo "Exiting without cleanup, --nocleanup specified"
```

Comment:
nitpick here: no dash in `--nocleanup` (should be `--no-cleanup` to mirror the option)

Reply:
whoops, I'll fix that up.